_rules/line-height-not-important-78fd32.md

```diff
@@ -21,32 +21,34 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element that is [visible][] and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
+This rule applies to any HTML element with [visible][] text that includes a [line-break](https://www.w3.org/TR/css-text-3/#line-breaking) and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
 
 ## Expectation
 
 For each test target, at least one of the following is true:
 
 - **not important**: the [computed][] value of its [line-height][] property is not [important][]; or
-- **large enough**: the [computed][] value of its [line-height][] property is not `normal`, and is at least `1.5` or 1.5 times the [computed][] value of its [font-size][] property; or
+- **large enough**: the [used](https://www.w3.org/TR/css-cascade-4/#used) value of its [line-height][] property is at least 1.5 times the [computed][] value of its [font-size][] property; or
 - **cascade**: the [cascaded][] value of its [line-height][] property is not a value [declared][] in its `style` attribute.
 
 ## Assumptions
 
 - There is no mechanism available on the page to adjust [line-height][]. If there is such a mechanism, it is possible to fail this rule while [Success Criterion 1.4.12 Text Spacing][sc1412] is still satisfied.
 
-- This rule assumes that when the [computed][] value of the [line-height][] property is `normal`, user agents chose a [used][] value below 1.5. [CSS recommendation][line-height normal] is to have a [used][] value between 1.0 and 1.2, thus too small to satisfy [Success Criterion 1.4.12 Text Spacing][sc1412].
+- The font size is constant for all text in the element. If font-size changes (e.g., through use of the `::first-line` pseudo-element) then the required line height would also change throughout the element. This is untested by the current rule. 
```

Comment:
This is definitely a weird assumption. The problem is that with an HTML element being the applicable "unit" we don't really want to say that the line height must be the 1.5 times the size of the largest font-size in the element. This would be really weird if the first line is very big, but all of the following lines are small.

Reply:
Yeah true. I'm not sure it makes much of a difference though.